contrib/win32/install/build/build-openssh.ps1

```diff
@@ -0,0 +1,136 @@
+[CmdletBinding()]
```

Comment:
We already have [openssh-build.ps1](https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/openssh/OpenSSH-build.ps1)  which does the build and package.

Please remove this.